<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <!-- Meta tags for social media banners, these should be filled in appropriatly as they are your "business card" -->
  <!-- Replace the content tag with appropriate information -->
  <meta name="description" content="DESCRIPTION META TAG">
  <meta property="og:title" content="SOCIAL MEDIA TITLE TAG"/>
  <meta property="og:description" content="SOCIAL MEDIA DESCRIPTION TAG TAG"/>
  <meta property="og:url" content="URL OF THE WEBSITE"/>
  <!-- Path to banner image, should be in the path listed below. Optimal dimenssions are 1200X630-->
  <meta property="og:image" content="static/image/your_banner_image.png" />
  <meta property="og:image:width" content="1200"/>
  <meta property="og:image:height" content="630"/>


  <meta name="twitter:title" content="TWITTER BANNER TITLE META TAG">
  <meta name="twitter:description" content="TWITTER BANNER DESCRIPTION META TAG">
  <!-- Path to banner image, should be in the path listed below. Optimal dimenssions are 1200X600-->
  <meta name="twitter:image" content="static/images/your_twitter_banner_image.png">
  <meta name="twitter:card" content="summary_large_image">
  <!-- Keywords for your paper to be indexed by-->
  <meta name="keywords" content="KEYWORDS SHOULD BE PLACED HERE">
  <meta name="viewport" content="width=device-width, initial-scale=1">


  <title>Arabidopsis Dataset</title>
  <link rel="icon" type="image/x-icon" href="static/images/icon.png">
  <link href="https://fonts.googleapis.com/css?family=Google+Sans|Noto+Sans|Castoro"
  rel="stylesheet">

  <link rel="stylesheet" href="static/css/bulma.min.css">
  <link rel="stylesheet" href="static/css/bulma-carousel.min.css">
  <link rel="stylesheet" href="static/css/bulma-slider.min.css">
  <link rel="stylesheet" href="static/css/fontawesome.all.min.css">
  <link rel="stylesheet"
  href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
  <link rel="stylesheet" href="static/css/index.css">

  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://documentcloud.adobe.com/view-sdk/main.js"></script>
  <script defer src="static/js/fontawesome.all.min.js"></script>
  <script src="static/js/bulma-carousel.min.js"></script>
  <script src="static/js/bulma-slider.min.js"></script>
  <script src="static/js/index.js"></script>
</head>
<body>



  <section class="hero">
    <div class="hero-body">
      <div class="container is-max-desktop">
        <div class="columns is-centered">
          <div class="column has-text-centered">
            <h1 class="title is-1 publication-title">A Multimodal Dataset on the Genome of the Model Plant <i>Arabidopsis thaliana</i> with Application Towards Hypergraph Learning</h1>
            <div class="is-size-5 publication-authors">
              <!-- Paper authors -->
              <span class="author-block">
                <!--<a href="FIRST AUTHOR PERSONAL LINK" target="_blank">Anonymous Authors</a>-->
                <a>Anonymous Author(s)</a>
              </span>
            </div>

            <!--<div class="is-size-5 publication-authors">
              <span class="author-block">NeurIPS 2024 Dataset and Benchmarks Track</span>
            </div>
            -->

            <div class="column has-text-centered">
              <div class="publication-links">
                    
                <!-- Arxiv PDF link -->
                <!--
                <span class="link-block">
                  <a href="https://arxiv.org/pdf/<ARXIV PAPER ID>.pdf" target="_blank"
                  class="external-link button is-normal is-rounded is-dark">
                    <span class="icon">
                      <i class="fas fa-file-pdf"></i>
                    </span>
                    <span>Paper (coming soon)</span>
                  </a>
                </span>
                -->

                <!-- Supplementary PDF link 
                <span class="link-block">
                  <a href="static/pdfs/supplementary_material.pdf" target="_blank"
                  class="external-link button is-normal is-rounded is-dark">
                    <span class="icon">
                      <i class="fas fa-file-pdf"></i>
                    </span>
                    <span>Supplementary</span>
                  </a>
                </span>
                -->

              <!-- Github link -->
              <!-- 
              <span class="link-block">
                <a href="https://github.com/anonymo-user/Video-ASD-Dataset/tree/master" target="_blank"
                class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                    <i class="fab fa-github"></i>
                  </span>
                  <span>Code</span>
                </a>
              </span>
              -->

              <!-- ArXiv abstract Link -->
              <!--
              <span class="link-block">
                <a href="https://arxiv.org/abs/<ARXIV PAPER ID>" target="_blank"
                class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                    <i class="ai ai-arxiv"></i>
                  </span>
                  <span>arXiv (coming soon)</span>
                </a>
              </span>
              -->

              <!-- Code link -->
              <!-- TODO: redo for Arabidopsis dataset codebase
              <span class="link-block">
                <a href="static/code/VideoASDRepo.zip" target="_blank"
                class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                    <i class="fas fa-code"></i>
                  </span>
                  <span>Code</span>
                </a>
              </span>
              -->

              <!-- CSV Annotations -->
              <!-- TODO: redo for Arabidopsis dataset metadata
              <span class="link-block">
                <a href="static/anns/csv_files.zip" target="_blank"
                class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                    <i class="fas fa-file-csv"></i>
                  </span>
                  <span>CSV Files</span>
                </a>
              </span>
              -->

              <!-- Text Facial Expression Labels -->
              <!-- TODO: redo for additional Arabidopsis dataset data
              <span class="link-block">
                <a href="static/anns/text_v1.zip" target="_blank"
                class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                    <i class="fas fa-user-tag"></i>
                  </span>
                  <span>Text Labels</span>
                </a>
              </span>
              -->

              <!-- Features -->
              <!-- TODO: redo for even more Arabidopsis dataset data
              <span class="link-block">
                <a href="https://drive.google.com/drive/folders/14328_XOly7ma5p6nkHGenA7ugkFmQAjJ?usp=sharing" target="_blank"
                class="external-link button is-normal is-rounded is-dark">
                  <span class="icon">
                    <i class="fas fa-file-video"></i>
                  </span>
                  <span>Features</span>
                </a>
              </span>
              -->
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>



<!-- Teaser video
<section class="hero teaser">
  <div class="container is-max-desktop">
    <div class="hero-body">
      <video poster="" id="tree" autoplay controls muted loop height="100%">
        <source src="static/videos/banner_video.mp4"
        type="video/mp4">
      </video>
      <h2 class="subtitle has-text-centered">
        Aliquam vitae elit ullamcorper tellus egestas pellentesque. Ut lacus tellus, maximus vel lectus at, placerat pretium mi. Maecenas dignissim tincidunt vestibulum. Sed consequat hendrerit nisl ut maximus. 
      </h2>
    </div>
  </div>
</section>
End teaser video -->



<!-- Hypergraphs and data showcasing in multiple figures -->
<section class="hero is-small">
  <div class="hero-body">
    <div class="container">
       <div class="item">
        <!-- Your image here -->
        <img src="static/images/graph-vs-hypergraph-2-caption.png" alt="graph-vs-hypergrah-2"/>
        <h2 class="subtitle has-text-centered">
          An illustration of representing a network of genes using simple graphs and hypergraphs. (A) Nodes representing individual genes contain information specific to the corresponding gene and are initially unconnected. (B) A simple graph, where, where edges can join any two nodes, is one method of connecting all the genes in <i>Arabidopsis thaliana</i>. (C) A hypergraph joins gene nodes based on each genes' known contribution to several biological functions. A hyperedge can join any number of nodes, and any node can be a member of several hyperedges at once since genes may impact to different functions of a biological system, hence the overlap of some nodes shown.
        </h2>
      </div>
</div>
</section>



<!-- Paper abstract -->
<section class="section hero is-light">
  <div class="container is-max-desktop">
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3">Abstract</h2>
        <div class="content has-text-justified">
          <p>
            The genome of an organism offers a wealth of information vital to understanding how it reacts to outside stimuli. One such model plant, called <i>Arabidopsis thaliana</i>, has been critical to developing biology researchers' understanding of other plants, including crop plants. Meanwhile, in recent years, hypergraphs and deep learning have seen substantial progress toward understanding datasets with higher-order relationships. Gene data and relationships within the genome fit well into the hypergraph structure, where its hyperedges naturally represent biological functions genes are known to contribute to. This is more intuitive than pair-wise edges in simple graphs, where connecting genes is not as straightforward. To the best of our knowledge, there is no work that connects hypergraphs and deep learning towards a complete set of genes as the nodes. To connect the biology and deep learning communities, we bring together different sources of gene and function information into one data package. We provide details over what we call the <i>Arabidopsis</i> dataset as well as go over transcriptomic data from <i>Arabidopsis</i> plants we have assembled. We then provide baseline experimental results to showcase how hypergraph models learn correlations amongst gene features to predict the up- or down-regulation with regards to gene expression. We further provide justification for using hypergraphs over graphs for this dataset. Finally, to address the challenges of our dataset, we discuss the experimental results and offer advice for future directions.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<!-- End paper abstract -->



<!-- Highlights as in the paper -->
<section class="section hero">
  <div class="container is-max-desktop">
    <div class="columns is-centered has-text-centered">
      <div class="column is-five-sixths">
        <h2 class="title is-3">Highlights</h2>
        <div class="content has-text-justified">
          <!-- Bullet points below <li>...</li> -->
          <ul>
            <li> We organize a dataset for gene information and relevant biological function information from multiple public sources concerning <i>Arabidopsis thaliana</i>, and then release in one package.</li>

            <li> On top of the gene information, we have assembled transcriptomic data using 24 mutant <i>Arabidopsis</i> plants or specimens.</li>
    
            <li> We conduct experiments using simple graph- and hypergraph-backbone and show that hypergraphs, with their more compact edge representation, are able to learn across connections of genes, while simple graphs struggle to do so.</li>
          </ul>
          
        </div>
      </div>
    </div>
  </div>
</section>



<!-- Our Features Showcase (w/ caption, Fig. 2) -->
<!-- First figure: hyperedge breakdown -->
<section class="hero is-small">
  <div class="hero-body">
    <div class="container">
       <div class="item">
        <!-- Your image here -->
        <img src="static/images/hyperedge-figure-5-caption.png" alt="hyperedge-figure-5"/>
        <h2 class="subtitle has-text-centered">
          A visualization of translating the biological functions the genes within <i>Arabidopsis thaliana</i> contribute to, and how such groups translate into the hypergraph structure. Shapes are for illustrative purposes.
        </h2>
      </div>
</div>
</section>

<!-- Second figure: gene features -->
<section class="hero is-small">
  <div class="hero-body">
    <div class="container">
       <div class="item">
        <!-- Your image here -->
        <img src="static/images/hypergraph-data-2-caption.png" alt="hypergraph-data-2"/>
        <h2 class="subtitle has-text-centered">
          An illustration of the multimodal data afforded by our <i>Arabidopsis</i> dataset. Genes have corresponding simple textual descriptions that indicate what their function is (along with identifying information, not shown here). Genes also have associated nucleotide sequences, which can be encoded for deep learning models. (There are also more sequences available, not shown here.) Finally, we add our transcriptomics data, where we measured gene expression levels for all genes across different plant specimens. In a preprocessing phase, each of these components are encoded or transformed which make up a row vector, which finally gives us the gene node features.
        </h2>
      </div>
</div>
</section>



<!-- Dataset Showcase -->
<!-- ...dataset statistics -->
<section class="section hero is-light">
  <div class="container is-max-desktop">
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3"><i>Arabidopsis</i> Dataset Statistics</h2>
        <div class="content has-text-justified">
          <p>
            The hypergraph data structure allows for a more compact representation compared to the simple graph.
          </p>

          <ul>
            <li><b>38,286</b> genes with text descriptions, gene nucleotide sequences, and transcriptomics features.</li>

            <li>There are <b>7,079</b> biological functions in <i>Arabidopsis</i> that genes contribute to. I.e., these naturally translate into <b>hyperedges</b>.</li>

            <li><b>149,864</b> total connections between all genes and all hyperedges.</li>

            <li><b>861,888</b> total edges between all genes for the simple graph when connecting top-10 most similar nodes. We run into memory and storage issues when connecting with more simple edges.</li>

            <li>We measured gene expression for <b>24</b> <i>Arabidopsis</i> plant specimens.</li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</section>

<!-- ...mutant groups from transcriptomics data -->
<section class="section hero is-light">
  <div class="container is-max-desktop">
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3">Transcriptomics Data</h2>
        <div class="content has-text-justified">
          <p>
            For the transcriptomic data, we have four mutant groups of plant specimens across the 24 plants, described here. Each of the four groups contains six plants. We focus on three individual genes--FAD7, EX1, and EX2.
          </p>

          <ul>
            <li><b>Col</b>: The control group; no mutations.</li>

            <li><b>FAD7</b>: Mutant in solely the FAD7 gene.</li>

            <li><b>EX1EX2</b>: Mutant in the EX1 and EX2 genes.</li>

            <li><b>FAD7EX1EX2</b>: Mutant in all three focus genes.</li>
          </ul>

          <p>
            We compare these four groups in six comparison scenarios. We measure how gene expression changes (goes up--upregulated, or goes down--downregulated, or no change) and train graph neural networks on these scenarios.
          </p>

          <ul>
            <li>EX1EX2 vs Col</li>

            <li>EX1EX2 vs FAD7</li>

            <li>FAD7EX1EX2 vs Col</li>

            <li>FAD7EX1EX2 vs EX1EX2</li>

            <li>FAD7EX1EX2 vs FAD7</li>

            <li>FAD7 vs Col</li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</section>



<!-- Experimental Results -->
<!-- ...Node classification (ALL) -->
<section class="hero is-small">
  <div class="hero-body">
    <div class="container">
       <div class="item">
        <!-- Your image here -->
        <img src="static/images/all-node-class-tables.png" alt="all-node-class-tables"/>
        <h2 class="subtitle has-text-centered">
          Experimental results for node classification comparing frameworks using a graph convolution backbone and another using a hypergraph convolution backbone. All models classify, given different transcriptomics data and additional gene features, whether a gene has been up- or down-regulated, or neither. We report classification accuracy, and then F1, precision, and recall averaged on all three categories.
        </h2>
      </div>
</div>
</section>



<!-- Data Preprocessing -->
<section class="hero is-small">
  <div class="hero-body">
    <div class="container">
       <div class="item">
        <!-- Your image here -->
        <img src="static/images/data-preprocessing-procedure-feb21.png" alt="data-preprocessing-procedure"/>
        <h2 class="subtitle has-text-centered">
          The data preprocessing procedure for transforming the raw data into the node features. Individual data is transformed into a sequence embedding (e.g., of text of nucleotides) or normalize data (e.g., transcriptomics) before concatenating into the final features. This features matrix is our input to all hypergraph models.
        </h2>
      </div>
</div>
</section>



<!-- Training Procedure -->
<section class="hero is-small">
  <div class="hero-body">
    <div class="container">
       <div class="item">
        <!-- Your image here -->
        <img src="static/images/training-procedure-feb21.png" alt="data-preprocessing-procedure"/>
        <h2 class="training-procedure">
          The training procedure. Some nodes are indexed for training while others are used to evaluate the model while all nodes aggregate information from their edge neighborhoods.
        </h2>
      </div>
</div>
</section>



<!-- Notes -->
<!-- TODO: add notes for Arabidopsis dataset work
<section class="section hero is-light">
  <div class="container is-max-desktop">
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3">Notes</h2>
        <div class="content has-text-justified">
          <p>
            <ul>
              <li>The links to download the code, csv files (for training and data retrieval), text labels (the face expression descriptions), and the features are at the top.</li>
            </ul>

            <ul>
              <li>We suggest everyone to read the supplementary material for further technical details of the dataset to understand the code + feature data + annotations.</li>
            </ul>
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
-->



<!-- Licensing -->
<!-- TODO: insert licensing for dataset here.  
<section class="section hero is-small">
  <div class="container is-max-desktop">
    <div class="columns is-centered has-text-centered">
      <div class="column is-four-fifths">
        <h2 class="title is-3">Licensing</h2>
        <div class="content has-text-justified">
          <p>
            The "Video ASD" dataset, its features and its annotations, are released under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0 license</a>. 
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
-->



<!-- References -->
<!-- TODO --> 



<!--BibTex citation -->
  <section class="section" id="BibTeX">
    <div class="container is-max-desktop content">
      <h2 class="title">BibTeX</h2>
      <pre><code>TBD</code></pre>
    </div>
</section>
<!--End BibTex citation -->



<footer class="footer">
<div class="container">
  <div class="columns is-centered">
    <div class="column is-8">
      <div class="content">
        <p>
          This page was built using the <a href="https://github.com/eliahuhorwitz/Academic-project-page-template" target="_blank">Academic Project Page Template</a> which was adopted from the <a href="https://nerfies.github.io" target="_blank">Nerfies</a> project page.
          You are free to borrow the of this website, we just ask that you link back to this page in the footer. <br> This website is licensed under a <a rel="license"  href="http://creativecommons.org/licenses/by-sa/4.0/" target="_blank">Creative
          Commons Attribution-ShareAlike 4.0 International License</a>.
        </p>
      </div>
    </div>
  </div>
</div>
</footer>



<!-- Statcounter tracking code -->
<!-- You can add a tracker to track page visits by creating an account at statcounter.com -->
<!-- End of Statcounter Code -->



</body>
</html>
